lib/oneview-sdk/resource/api300/c7000/enclosure_group.rb

```diff
@@ -26,11 +26,64 @@ def initialize(client, params = {}, api_ver = nil)
           # Default values:
           @data['type'] ||= 'EnclosureGroupV300'
           @data['stackingMode'] ||= 'Enclosure'
+          @data['enclosureCount'] ||= 1
           @data['interconnectBayMappingCount'] ||= 8
-          create_interconnect_bay_mapping unless @data['interconnectBayMappings']
           super
         end
 
+        # Adds the logical interconnect group
+        # @param [OneviewSDK::LogicalInterconnectGroup] lig Logical Interconnect Group
```

Comment:
I've updated the return statements, but left these as-is because technically it can accept a `OneviewSDK::API200::LogicalInterconnectGroup`, `OneviewSDK::API300::C7000::LogicalInterconnectGroup`, or `OneviewSDK::API300::Synergy::LogicalInterconnectGroup` as a parameter. I wanted to keep it generic